### PR TITLE
nmap 7.94

### PR DESCRIPTION
**Pull request by chenrui333**
Created by `brew bump`

---

Created with `brew bump-formula-pr`.

### CODE REVIEW

**Comment by Porkepix** [comment]
Wanted to open it earlier, but archive was not yet available; I was not sure about letting the lua comment here, and also I think the part with the openssl flag isn't needed anymore. `openssl@3` might even work?

**Comment by FtZPetruska** [comment]
> `openssl@3` might even work?

According to their [changelog](https://nmap.org/changelog.html), `openssl@3` should be supported on >=7.93

**Comment by SMillerDev** [comment]
```
  ./nse_main.h:49:21: error: unknown type name 'lua_State'
  void nse_gettarget (lua_State *, int);
                      ^
  10 errors generated.
  make: *** [FingerPrintResults.o] Error 1
```

**Comment by carlocab** [comment]
https://github.com/nmap/nmap/issues/2648

File: Formula/nmap.rb [hunk 1]
```diff
@@ -62,6 +49,9 @@ def install
     ]
 
     system "./configure", *args
+    # Workaround for "./nse_main.h:25:26: error: unknown type name 'lua_State'"
+    # TODO: Report this upstream.
+    ENV.append_to_cflags "-DHAVE_LUA5_4_LUA_H"
```
Comment (carlocab) [inline Formula/nmap.rb:54]:
```suggestion
    # Workaround for "./nse_main.h:25:26: error: unknown type name 'lua_State'"
    # TODO: Report this upstream.
    ENV.append_to_cflags "-DHAVE_LUA5_4_LUA_H"
    system "./configure", *args
```

File: Formula/nmap.rb [hunk 2]
```diff
@@ -62,6 +49,9 @@ def install
     ]
 
     system "./configure", *args
```
Comment (carlocab) [inline Formula/nmap.rb:51]:
```suggestion
```

**Review by SMillerDev — DISMISSED** [review]

**Pull request closed without merging**